projects/plugins/backup/src/php/class-jetpack-backup.php

```diff
@@ -232,6 +241,31 @@ public static function get_backup_capabilities() {
 		);
 	}
 
+	/**
+	 * Gets information about the currently promoted backup product.
+	 *
+	 * @return string|WP_Error A JSON object of the current backup product being promoted if the request was successful, or a WP_Error otherwise.
+	 */
+	public static function get_backup_promoted_product_info() {
+		$request_url   = 'https://public-api.wordpress.com/rest/v1.1/products?_locale=' . get_user_locale() . '&type=jetpack';
```

Comment:
Should this currently return non-USD prices if `get_user_locale` is returning say `en_GB` or `es_MX`? Seemed to always display USD on the plugin's page when I tried changing the locale.

Reply:
Looks like there are two issues here.  The param should be `locale` and not `_locale`, and that's just for translation of descriptions the API gives back.  There is also a `currency` param that can be set to something like `USD` or `EUR` etc, but I need to figure out how to get that information.  I think we have it on already connected accounts, but this is in use before the connection is established so I'll have to find another way, if one exists, on the site to determine currency.